core/modules/layout/tests/layout.test

```diff
@@ -1523,7 +1596,7 @@ class LayoutInterfaceTest extends BackdropWebTestCase {
     $result = config_get('layout.layout.' . $second_layout_name, 'name');
     $this->assertEqual($result, $second_layout_name, 'Config created for second layout.');
     $result = config_get('layout.menu_item.' . $second_layout_name, 'name');
-    $this->assertNull($result, 'No config created for second layout menu item (reuses the existing one).');
+    $this->assertNULL($result, 'No config created for second layout menu item (reuses the existing one).');
```

Comment:
These renamings should be reverted. The method is correctly named `assertNull`. I'm guessing this was accidental in a find/replace all.